migrations/000001_schema.up.sql

```diff
@@ -2,10 +2,8 @@
 -- PostgreSQL database dump
```

Comment:
We need to come up with a better method for coordinating DB changes. I also have a change pending and either merge will result in a huge conflict here. Any clever ideas?

Reply:
1 person can be responsible for directus changes (changes to the schema.yml), so any changes must be coordinated with that person.
Anything directus doesn't manage, like db views, can be done via directus migrations.
- This makes the psql dump inconsistent, but just do a new dump during merge.